libclc/utils/libclc-remangler/LibclcRemangler.cpp

```diff
@@ -951,8 +927,8 @@ class LibCLCRemangler : public ASTConsumer {
   }
 
 private:
-  ASTContext *ContextAST;
-  LLVMContext ContextLLVM;
+  ASTContext *AST;
+  LLVMContext Ctx;
```

Comment:
I don't agree that this change improves readability. `ATS` name is confusing. I think the name must reflect that it's an ATS context and not AST.
```suggestion
  ASTContext *ASTCtx;
  LLVMContext LLVMCtx;
```

Reply:
That makes sense. A quick grep of clang shows they use `Ctx` or `Context`, but given there are two different contexts here they need disambiguating.